### ISSUE TITLE
Pen tilt angle is not in radians

### ISSUE THREAD

**Issue by celyk** [godotengine/godot#85990]
### Tested versions

4.2.stable

### System information

Windows 11

### Issue description

I'm trying to get a 3D vector pointing in the direction of the tablet pen using [InputEventMouseMotion.tilt](https://docs.godotengine.org/en/stable/classes/class_inputeventmousemotion.html#class-inputeventmousemotion-property-tilt). 

Based on [this Windows Ink reference](https://learn.microsoft.com/en-us/uwp/api/windows.ui.input.inking.inkpoint.tilty?view=winrt-22621#windows-ui-input-inking-inkpoint-tilty) for `TiltX` and `TiltY`, I should be able to do something like 
`Vector3(tan(event.tilt.x), tan(event.tilt.y), 1)`. This doesn't work because the angle is not given in degrees nor radians, but normalized to a [-1,1] range instead.

https://github.com/godotengine/godot/blob/b94eb58d35b3dd8a9f522bc90df0db73862ef326/platform/windows/display_server_windows.cpp#L3371

I'm not saying that it can't work, but it's an odd choice for something that can be physically measured. I have also found a mistake in the Android input handler. It seems that the sine of the angle is being returned rather than the angle itself.
https://github.com/godotengine/godot/blob/b94eb58d35b3dd8a9f522bc90df0db73862ef326/platform/android/java/lib/src/org/godotengine/godot/input/GodotInputHandler.java#L478-L485

I doubt the two versions are collinear. Sadly, I don't have an Android device to confirm this.

### Steps to reproduce

Running the MRP displays a visual indicator of the pen, and the horizontal tilt angle in degrees. I currently have to remap the angle from [-1,1] to [-π/2, π/2] for the correct result, as I explained. It should roughly line up with the actual rotation of the tablet pen if tilt is supported.

On Android, you should be able to produce the bug by tilting the pen by 45 degrees, and it clearly not lining up. Perhaps an emulator can be used for thorough testing.

### Minimal reproduction project (MRP)

[tilt-MRP.zip](https://github.com/godotengine/godot/files/13626842/tilt-MRP.zip)

**Comment by AThousandShips** [comment]
This range is intended and [documented](https://docs.godotengine.org/en/stable/classes/class_inputeventmousemotion.html#class-inputeventmousemotion-property-tilt), how other platforms and APIs implement this isn't relevant to how Godot does it, changing this would break compatibility so isn't an option, perhaps a note could be added but the conversion is obvious and the actual value range is stated in the documentation

**Comment by celyk** [comment]
That's fair. I don't mind it staying as is so long as it's consistent across platforms. I definitely would put a note in there about converting to radians, because the current description is not comprehensive. If the conversion were obvious, then the apparent mistake on Android would not be.

**Comment by AThousandShips** [comment]
Are you sure it's a mistake? It multiplies the orientation of the pen direction by the tilt, this should be an ellipse right?

**Comment by celyk** [comment]
Yeah, I'm certain it's different. I recognize it as the conversion formula from spherical to cartesian coordinates. Those aren't angles, they're distance from the x and y axis. If tilt were in that form already, I could just use Pythagoras to get my final vector.

 Simple cases reveal that the vectors are usually different in magnitude. For instance, what if the pen is only tilted 45 degrees horizontally? Then, theoretically:

On Windows
`event.tilt.x == 0.5`
On Android
`event.tilt.x == sin(pi/4)`

**Comment by bruvzg** [comment]
> angle is not given in degrees nor radians, but normalized to a [-1,1] range instead.

For the reference: tilt was selected to be `-1...1` range because macOS use this format (and it was first implemented for macOS): [This property’s value can range from -1.0 to 1.0 for both axes](https://developer.apple.com/documentation/appkit/nsevent/1534226-tilt?language=objc).

On Windows, it's (WinInk) [The angle of tilt of the pointer along the y-axis in a range of -90 to +90](https://learn.microsoft.com/en-us/windows/win32/api/winuser/ns-winuser-pointer_pen_info) or (WinTab) ["azimuth" + "altitude"](https://developer-docs.wacom.com/docs/icbt/windows/wintab/wintab-reference/#orientation).

On iOS, it's also ["azimuth" + "altitude"](https://developer.apple.com/documentation/uikit/touches_presses_and_gestures/illustrating_the_force_altitude_and_azimuth_properties_of_touch_input?language=objc).

**Comment by celyk** [comment]
I've finally been able to test this on my iPad. If I'm right it should incorrectly display 63 degrees When the pen is at a 45 degree angle. Since `sin(pi/4) * pi/2 radians ≈ 63.63961 degrees`

I look from the side and try to hold the pen at 45 degree tilt, and sure enough, it tells me 63 degrees. I'm just eye balling it but it's consistent.
